Tests/Responses/ListUserReceiptResponse.json

```diff
@@ -0,0 +1,54 @@
+{
+    "count" : 3,
+    "offset" : 0,
+    "limit" : 10,
+    "data" : [ {
+        "journalId" : "51660665",
+        "type" : "PAYMENT",
+        "createdOn" : "2019-05-27T15:42:07",
+        "entry" : "CREDIT",
+        "sourceToken" : "act-b1f6dc28-e534-45f4-a661-3523f051f77a",
```

Comment:
Can you use some dummy token instead of the real ones? Thank you.

Reply:
np, fixed